packages/template-ui/src/components/ChannelHeader.vue

```diff
@@ -78,7 +81,18 @@ export default {
   },
   computed: {
     ...mapState(['displayLogoInHeader', 'hasDarkHeader']),
-    ...mapGetters(['headerTitle', 'headerDescription']),
+    sectionOrChannel() {
+      if (this.section === undefined || this.section.id === this.channel.id) {
```

Comment:
Is this second expression needed? If the section and channel id are the same it should  be the same to return `this.channel` or `this.section`. In that case we can replace this whole method to `return this.section || this.channel;`

Reply:
Yes, the second expression is needed. What this does is return the channel information in case the section (topic) is the root one, in which case the ID will match with the channel.

If we remove the second part, navigating to `/t/ROOT_ID` would have a different header information than navigating to `/`.

All I'm doing is maintaining the old behavior, so I prefer to keep it as-is.